src/components/application_manager/src/commands/hmi/vi_unsubscribe_vehicle_data_request.cc

```diff
@@ -44,7 +44,6 @@ VIUnsubscribeVehicleDataRequest::~VIUnsubscribeVehicleDataRequest() {}
 
 void VIUnsubscribeVehicleDataRequest::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
-
```

Comment:
@IGapchuk not related changes

Reply:
@AKalinich-Luxoft  Revert in 36b0eadff97e5756477026c3d51f33e575f5d77a.